libraries/stdlib/genmsl/stdlib_genmsl_impl.mtlx

```diff
@@ -41,10 +41,6 @@
     <input name="default" type="vector4" implname="default_value" />
   </implementation>
 
-  <!-- <normalmap> -->
```

Comment:
Instead of removing these implementations, don't they just need to be redirected to the GLSL implementation? 

Otherwise for a `genmsl` target - how does it find the correct implementation at shader generation time?

Reply:
Our MSL shader generator already inherits from the GLSL shader generator, so this is handled automatically!

Reply:
I didn't realize this... maybe that means there are other lines in this file that aren't necessary - as they are just referencing GLSL implementations.

Reply:
I've been working through much of the unneeded MSL/GLSL duplication this week, though additional improvements can likely still be found!

For reference, here's where the inheritance of genmsl from genglsl is declared:

https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/libraries/targets/genmsl.mtlx